app/models/viewmodels/notificationspage/MovementTotalsResponseConverter.scala

```diff
@@ -39,8 +39,10 @@ class MovementTotalsResponseConverter @Inject()(decoder: Decoder, dateTimeFormat
     if (canConvertFrom(notification)) {
 
       val crcCodeExplanation = notification.crcCode.flatMap(buildCrcCodeExplanation)
-      val roeCodeExplanation = notification.masterRoe.flatMap(buildRoeCodeExplanation)
-      val soeCodeExplanation = notification.masterSoe.flatMap(buildSoeCodeExplanation)
+      val roeCodeExplanation =
+        notification.entries.headOption.flatMap(_.entryStatus).flatMap(_.roe).flatMap(buildRoeCodeExplanation)
```

Comment:
Have I understood correctly?  There can only ever be max one Entry?   In which case, why is it stored as a Seq?

Reply:
According to the schema, there might be several Entries. Even in our case, for association we will have MUCR and DUCR Entries. The reason it is only processing the first Entry is not to change the behaviour. It will be changed in the next PR as part of the ticket.